modules/openapi-generator/src/main/resources/swift5/modelObject.mustache

```diff
@@ -1,5 +1,5 @@
-{{^objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} {{#useClasses}}final class{{/useClasses}}{{^useClasses}}struct{{/useClasses}} {{{classname}}}: {{#useVapor}}Content{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}{{#vendorExtensions.x-swift-hashable}}, Hashable{{/vendorExtensions.x-swift-hashable}} {
-{{/objcCompatible}}{{#objcCompatible}}@objc {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} class {{classname}}: NSObject, Codable {
+{{^objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} {{#useClasses}}final class{{/useClasses}}{{^useClasses}}struct{{/useClasses}} {{{classname}}}: {{#useVapor}}Content{{/useVapor}}{{^useVapor}}Codable, JSONEncodable{{/useVapor}}{{#vendorExtensions.x-swift-hashable}}, Hashable{{/vendorExtensions.x-swift-hashable}}{
+{{/objcCompatible}}{{#objcCompatible}}@objc {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} class {{classname}}: NSObject, Codable, JSONEncodable {
 {{/objcCompatible}}
 
```

Comment:
Could you please also add the `JSONEncodable` conformance after `Codable` in the following files, please?
modelEnum.mustache
modelInlineEnumDeclaration.mustache
modelOneOf.mustache

Reply:
Done with https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/d855a09caf472aa3d733317cfe607ef7b80295c5 and  https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/4929d5bbf87cd3b222677d2346d499eb1f244a0b .